### PR TITLE
Feat: new Atlas startup code to include "team project" selection

### PR DESCRIPTION
**Pull request by pieterlukasse**
Jira Ticket: [VADC-712](https://ctds-planx.atlassian.net/browse/VADC-712)

### New Features
- Adjust Atlas starter to first display list of "team project" options available to the user, before opening Atlas

[VADC-569]: https://ctds-planx.atlassian.net/browse/VADC-569?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[VADC-712]: https://ctds-planx.atlassian.net/browse/VADC-712?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by pieterlukasse** [comment]
TODOs:
- storybook
- enable "full screen" option...somehow it is missing now

File: src/Analysis/OHDSIAtlas/Views/AtlasStarter.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        <div className='spinner-container'>
+          <Spin /> Retrieving the list of team projects.
+          <br />
+          Please wait...
+        </div>
+      </React.Fragment>
+    );
+  }
+  if (status === 'error') {
+    return 'Error';
```
Comment (jarvisraymond-uchicago) [inline src/Analysis/OHDSIAtlas/Views/AtlasStarter.jsx:33]: Validate this works in Storybook, consider showing user more descriptive error message

File: src/Analysis/OHDSIAtlas/Views/AtlasStarter.jsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  }
+  return (
+    <React.Fragment>
+      Select a team project:
+      <SelectTeamProjectDropDown
+        teamProjects={data.teams}
+        setSelectedTeamProject={setSelectedTeamProject}
+      />
+      <button
```
Comment (jarvisraymond-uchicago) [inline src/Analysis/OHDSIAtlas/Views/AtlasStarter.jsx:42]: Consider making AntD button

File: src/Analysis/OHDSIAtlas/Components/SelectTeamProjectDropDown.jsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+      id={'input-selectTeamProjectDropDown'}
+      showSearch
+      labelInValue
+      onChange={onChange}
+      placeholder='-select one of the team projects below-'
+      fieldNames={{ label: 'teamName', value: 'teamName' }}
+      options={teamProjects}
+      dropdownStyle={{ width: '300px' }}
+      style={{ width: '300px' }}
+      filterOption={(searchTerm, option) => (option?.teamName ?? '')
```
Comment (jarvisraymond-uchicago) [inline src/Analysis/OHDSIAtlas/Components/SelectTeamProjectDropDown.jsx:24]: Do we need filter code here?

**Review by jarvisraymond-uchicago — APPROVED** [review]
Looks good to me!

File: src/Analysis/OHDSIAtlas/Components/SelectTeamProjectDropDown.jsx (hunk 3 above)
Comment (pieterlukasse) [reply src/Analysis/OHDSIAtlas/Components/SelectTeamProjectDropDown.jsx]: good point. Removed.

**Pull request merged**